remoteworkitem/trackeritem_repository_test.go

```diff
@@ -0,0 +1,106 @@
+package remoteworkitem
+
+import (
+	"fmt"
+	"testing"
+
+	"golang.org/x/net/context"
+
+	"github.com/almighty/almighty-core/models"
+	"github.com/almighty/almighty-core/resource"
+	"github.com/almighty/almighty-core/test"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConvert(t *testing.T) {
+	resource.Require(t, resource.Database)
+
+	// Setting up the dependent tracker query and tracker data in the Database
+	db.Exec(`DELETE FROM "tracker_items"`)
+	tr := Tracker{URL: "https://api.github.com/", Type: ProviderGithub}
+	db.Create(&tr)
+	tq := TrackerQuery{Query: "some random query", Schedule: "0 0 0 * * *", TrackerID: tr.ID}
+	db.Create(&tq)
+	t.Log("Created Tracker Query and Tracker")
+
+	ts := models.NewGormTransactionSupport(db)
+
+	ts.Begin() // TODO: will have to put this inside transaction.do{}
```

Comment:
- Added the TODO
- Added the multiple transactions because each transaction is a test case in itself. The first transaction is needed to allow the second test case to do its job.